response_operations_ui/logger_config.py

```diff
@@ -32,7 +34,18 @@ def add_service(logger, method_name, event_dict):  # pylint: disable=unused-argu
         event_dict['service'] = service_name
         return event_dict
 
+    def zipkin_ids(logger, method_name, event_dict):
+        event_dict['zipkin_trace_id'] = ''
```

Comment:
don't think you need `event_dict['zipkin_trace_id'] = ''` and `event_dict['zipkin_span_id'] = ''`